academy/0.0-B9lab-Blockchains/5_crypto.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Cryptography"
+order: 3
+description: Cryptographic Foundations
+tag: fast-track
+---
+
+# Fundamentals
```

Comment:
Only include a headline level if there is at least one sentence between headlines. Please include an introductory sentence after this main headline.

Reply:
I believe I found a solution. Please recheck the headlines in my commit.